racket/collects/racket/private/pre-base.rkt

```diff
@@ -200,6 +200,8 @@
              (rename new-prop:procedure prop:procedure)
              (rename #%app #%plain-app)
              (rename lambda #%plain-lambda)
+             (rename λ #%plain-λ)
```

Comment:
I'm not sure if it should be `(rename λ #%plain-λ)` or `(rename lambda #%plain-λ)`.